inspec.yml

```diff
@@ -17,30 +17,25 @@ inputs:
     description: 'The username MSSQL DB Server'
```

Comment:
I'll update the branch name in the `depends.url` when the PR for the baseline has been approved and merged. Thanks!